backend/endpoints/sockets/scan.py

```diff
@@ -129,133 +116,9 @@ async def scan_platforms(
-
-                scanned_save.emulator = fs_emulator
-
-                if rom:
-                    scanned_save.rom_id = rom.id
-                    db_save_handler.add_save(scanned_save)
-
-            # Scanning states
-            fs_states = fs_asset_handler.get_assets(
-                platform.fs_slug, rom.file_name_no_ext, Asset.STATES
-            )
-            if len(fs_states) > 0:
-                log.info(f"\t · {len(fs_states)} states found")
-
-            for fs_emulator, fs_state_filename in fs_states:
-                scanned_state = scan_state(
-                    file_name=fs_state_filename,
-                    platform_slug=platform.fs_slug,
-                    emulator=fs_emulator,
-                )
-
-                state = db_state_handler.get_state_by_filename(
-                    rom.id, fs_state_filename
-                )
-                if state:
-                    # Update file size if changed
-                    if state.file_size_bytes != scanned_state.file_size_bytes:
-                        db_state_handler.update_state(
-                            state.id, {"file_size_bytes": scanned_state.file_size_bytes}
-                        )
-
-                    continue
-
-                scanned_state.emulator = fs_emulator
-
-                if rom:
-                    scanned_state.rom_id = rom.id
-                    db_state_handler.add_state(scanned_state)
-
-            # Scanning screenshots
-            fs_screenshots = fs_asset_handler.get_assets(
-                platform.fs_slug, rom.file_name_no_ext, Asset.SCREENSHOTS
-            )
-            if len(fs_screenshots) > 0:
-                log.info(f"\t · {len(fs_screenshots)} screenshots found")
-
-            for _, fs_screenshot_filename in fs_screenshots:
-                scanned_screenshot = scan_screenshot(
-                    file_name=fs_screenshot_filename, platform_slug=platform.fs_slug
-                )
-
-                screenshot = db_screenshot_handler.get_screenshot_by_filename(
-                    fs_screenshot_filename
-                )
-                if screenshot:
-                    # Update file size if changed
-                    if screenshot.file_size_bytes != scanned_screenshot.file_size_bytes:
-                        db_screenshot_handler.update_screenshot(
-                            screenshot.id,
-                            {"file_size_bytes": scanned_screenshot.file_size_bytes},
-                        )
-                    continue
-
-                if rom:
-                    scanned_screenshot.rom_id = rom.id
-                    db_screenshot_handler.add_screenshot(scanned_screenshot)
-
-            db_save_handler.purge_saves(rom.id, [s for _e, s in fs_saves])
-            db_state_handler.purge_states(rom.id, [s for _e, s in fs_states])
-            db_screenshot_handler.purge_screenshots(
-                rom.id, [s for _e, s in fs_screenshots]
-            )
```

Comment:
no longer need to scan for saves/states since they're loaded from the api. i might add this back when we build device sync, since it might make more sense at that point.

Reply:
I'm not sure if I am understanding this correctly, if we don't scan them how are we going to write them in the database?

Reply:
Right now the only way to add saves/states is by uploading them in the browser (api), or saving them when using emulatorjs (also api). Given the complexity of the save path we need to use to avoid collisions (ex `/assets/users/557365723a31/states/n64/mupen64plus/Super Mario 64.srm`), I think forcing users to upload saves via the UI makes the most sense.

When we build device sync, we can add a new scanner and watcher for the assets folder, but that all depends on how we implement sync.